### DISCUSSION TITLE
When the allocated memory space is smaller than the data size, more memory is not better .

### DISCUSSION THREAD

**Discussion by shatealaboxiaowang** [milvus-io/milvus#7106] [Q&A and General discussion]
When the allocated memory space is smaller than the data size, more memory is not better .
exp: data_size=20g, search_speed(cache_size=19g)=search_speed(cache_size=1G)

code:
myMilvus.set_config('cache', 'cache_size', '1GB')
top_k = int(confs_milvus.get('topk'))
q_records = [[random.random() for _ in range(dim)] for _ in range(1)]
for i in range(5):
start_time = time.time()
results = myMilvus.search(collection_name_create, top_k, q_records)
logging.info('Search running time: %s' % str(time.time() - start_time))

**Comment by yhmo** [comment]
This is a known issue: https://github.com/milvus-io/milvus/issues/2123
We think it is not a critical issue and did't fix it.
